app/js/app/directives/EventBookingForm.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2015 Stephen Cummins
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ * 		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+define([], function() {
+
+
+	return ["api", "$rootScope", function(api, $rootScope) {
+
+		return {
+			scope: true,
+
+ 			restrict: "A",
+			
+			templateUrl: "/partials/event_booking_form.html",
+
+			link: function(scope, element, attrs) {
+
+				scope.editingSelf = scope.targetUser._id == scope.user._id
+
+				scope.makeBooking = function(eventId, userId, additionalInformation){
```

Comment:
This method is never called, by any code that I can find.

Admins use a different method called `bookUserOnEvent(...)`, and users booking themselves use `requestBooking()`. What is it supposed to do? Is it obsolete?

Reply:
You're right I will remove this.